create_schema.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+db_name=$1
+
+createdb $db_name
```

Comment:
Do we want to check whether the database exists before creating it? To avoid the below error.

"createdb: database creation failed: ERROR:  database "ft_presentation" already exists"